### ISSUE TITLE
Refine preliminary prediction

### ISSUE THREAD

**Issue by alyoshenka** [itsjpb/470-avalanche-prediction#54]
I was able to put in some data and spit out a number. It has a 30% accuracy. Try to make that better.
- [x] Start by adding more data - longer timeframe
- [x] Then add more data - more weather sites
- [x] Make list of additional data points we can add
    - ex: was there sustained heavy snowfall? was there rapid warming? -> look up avalanche key things to watch for

[Hyperparameter Tuning](https://towardsdatascience.com/hyperparameter-tuning-the-random-forest-in-python-using-scikit-learn-28d2aa77dd74)

**Comment by alyoshenka** [comment]
Depends on #57

**Comment by alyoshenka** [comment]
Didn't work out as well as I'd hoped. Resolving in favor of a more detailed issue with teammate help.
